infrastructure-playbooks/purge-container-cluster.yml

```diff
@@ -532,6 +539,7 @@
 
     - name: remove ceph container image
       command: "{{ container_binary }} rmi {{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}"
+      failed_when: false
```

Comment:
Why are you adding the `failed_when` arg here ?

Reply:
I am thinking that if the local image of ceph does not exist, an error will occur here.

Reply:
This part was removed in the latest commit because I previously considered avoiding errors in the absence of container images